modules/core/client/app/config.js

```diff
@@ -1,23 +1,29 @@
-'use strict';
+(function (window) {
```

Comment:
I don't think it can be, unless angular is injecting it into a controller/service.